test/verify/check-system-info

```diff
@@ -494,7 +494,7 @@ class TestSystemInfo(MachineCase):
             b.enter_page('/system/hwinfo')
 
             if not expect_link_present:
-                b.wait_visible('#hwinfo th:contains(CPU)')
+                b._wait_present('#hwinfo th:contains(CPU)')
```

Comment:
Here the old code should actually work, as the next line makes sure that the "CPU security" line does *not* exist.